examples/Example.h

```diff
@@ -436,6 +436,11 @@ struct WgWindow : Window
         //Set the canvas target and draw on it.
         verify(canvas->target(instance, surface, width, height));
     }
+
+    void refresh() override
+    {
+        wgpuSurfacePresent(surface);
```

Comment:
This sync change must be addressed separately in https://github.com/thorvg/thorvg/pull/2490. Please make a commit for blending support first.